### ISSUE TITLE
about classifier-balancing

### ISSUE THREAD

**Issue by alsehaimaa** [pengzhiliang/FC_Mnist#1]
Hi
sorry for contacting you in this place as I did not find your email I am trying to use the loss function of Decoupling Representation and Classifier for Long-Tailed Recognition for my own data but I am not sure how to optimize. what is the right way?
what i did that i define the loss as:
Facebookloss=DiscCentroidsLoss(6,2048).to(device)
and in each epoch i calculate:
loss=Facebookloss(feat,actions_lable )
loss.backward()

Is it right?
